hardhat.config.ts

```diff
@@ -52,6 +52,12 @@ const config: HardhatUserConfig = {
             forking: {
                 url: `https://eth-mainnet.alchemyapi.io/v2/${alchemyApiKey}`,
             },
+            accounts: {
+                // This will result in 10000 ETH being deposited into this wallet on the local testnet
+                // Obviously don't use this on mainnet
+                mnemonic:
+                    "subway bright wet favorite tide muscle alley marble must ecology result treat",
```

Comment:
ser

Reply:
It's a completely random wallet - not mine.

Reply:
We need to enable the developer to have ETH deposited into their wallet. Is there a different approach you'd suggest? Let the dev provide the seed phrase as an env variable and have that imported into this?

Reply:
yup, we have a .env.example that has a template for a .env file. if you want to work off of that. Can the method for getting ETH deposited into the wallet work off of Private keys?

Reply:
Have removed the commits. Can you delete the comment so the preview also disappears?